src/validators/maxfee.validator.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import {ExpectedError, ValidationContext, Validator} from 'clime';
+
+export class MaxFeeValidator implements Validator<number> {
+    validate(value: number, context: ValidationContext): void {
+        if (!/^\d+$/.test(value.toString())) {
```

Comment:
I would prefer to avoid regex. How about ``Number.isInteger(value) && value >= 0`` or ``(value % 1 === 0) && (value >= 0)``?

Reply:
ok , I fix it